src/index.html

```diff
@@ -10,8 +10,177 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset class="personalInfo field">
+        <legend>Personal information</legend>
+        <div class="surname input">
+          <label class="surnameField">
+            Surname:
+            <input 
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="name input">
+          <label class="nameField">
+            Name:
+            <input 
+              type="text" 
+              name="name"                
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="age input">
+          <label class="ageField">
+            How old are You?
+            <input 
+              type="number" 
+              name="age" 
+              value="12" 
+              min="1" 
+              max="100"
+            >
+          </label>
+        </div>
+        <div class="date input">
+          <label class="dateField">
+            Full date of birth:
+            <input
+              type="date" 
+              name="date" 
+            >
+          </label>
+        </div>
+        <div class="term input">
+          <label class="termField">
+            I accept the term of the agreement
+            <input 
+              type="checkbox" 
+              name="term"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="registration field">
+        <legend>Registration</legend>
+        <div class="email input">
+          <label class="emailField">
+            E-mail:
+            <input 
+              type="email" 
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="password input">
+          <label class="passwordField">
+            Password:
+            <input 
+              type="text" 
+              name="password"
+              minlength="8" 
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="facts field">
+        <legend>An interesting fact about you!</legend>
+        <div class="cats input">
+          <label class="catsField" for="cats">
+            Do you love cats?
+            <input 
+              type="radio" 
+              name="cats"
+              id="catsFieldYes"
+              value="Yes"
+            >
+            <label for="catsField">Yes</label>
```

Comment:
Id's should be the same
```suggestion
              id="catsFieldYes"
              value="Yes"
            >
            <label for="catsFieldYes">Yes</label>
```